hack/buildah-vendor-treadmill

```diff
@@ -503,7 +536,9 @@ END_QUERY
     print $postquery, "\n"            if $debug;
     my $res = $ua->post($API_URL, Content => $postquery);
     if ((my $code = $res->code) != 200) {
-        print $code, " ", $res->message, "\n";
+        warn "$ME: GraphQL request failed on $API_URL:\n";
+        print STDERR "  ", $code, " ", $res->message, "\n";
+        warn "Cannot continue.\n";
```

Comment:
If we want to be really nice here, check for 401 and then say you have to create a GITHUB_TOKEN and link to https://github.com/settings/tokens and then `export GITHUB_TOKEN=<mytoken>` and rerun the script.

I don't know why I need a token but this would help other to figure it without having to read the script.

Reply:
@Luap99 I had hoped to do that, but didn't know enough about the symptom. I'll add a check for `410`, but before I re-push, is there any chance that you still have a screen capture of the error?

Reply:
I don't have a capture but I am sure the only output after `HEAD is a buildah vendor commit. Good.` was `401 unauthorized` and the script exited.

Reply:
Resubmitted, but differently. Turns out, it wasn't your IP, it's just github rejecting that endpoint entirely if called without auth. I reworked the script so if you don't have `$GITHUB_TOKEN` set it will fail, suggest that you set it, or if you don't have one, offer a URL to find out the PR ID and reinvoke the script. More than I would normally add in a PR like this, but hey, nobody's reviewing this anyway, so I can get away with anything :-)